pkg/fission-cli/util/util.go

```diff
@@ -372,6 +372,20 @@ func UpdateMapFromStringSlice(dataMap *map[string]string, params []string) bool
 	return updated
 }
 
+// UpdateEnvVarFromStringSlice parses key, val from "key=val" string array and updates passed []v1.EnvVar
+func UpdateEnvVarFromStringSlice(envVar []v1.EnvVar, params []string) bool {
+	updated := false
+	for idx, m := range params {
+		keyValue := strings.SplitN(m, "=", 2)
+		if len(keyValue) == 2 {
```

Comment:
Since we have mentioned  2 as an argument to `SplitN` function.
It will return 2 or <2 no of strings. We should have validation accordingly both for success/failure case.
Please add unit tests so that this gets validated.